lib/app/modules/alarmChallenge/views/alarm_challenge_view.dart

```diff
@@ -3,20 +3,30 @@ import 'package:get/get.dart';
 import 'package:ultimate_alarm_clock/app/modules/alarmChallenge/views/maths_challenge_view.dart';
 import 'package:ultimate_alarm_clock/app/modules/alarmChallenge/views/qr_challenge_view.dart';
 import 'package:ultimate_alarm_clock/app/modules/alarmChallenge/views/shake_challenge_view.dart';
+import 'package:ultimate_alarm_clock/app/services/haptic_feedback_service.dart';
 import 'package:ultimate_alarm_clock/app/utils/constants.dart';
 
 import '../controllers/alarm_challenge_controller.dart';
 
 class AlarmChallengeView extends GetView<AlarmChallengeController> {
-  const AlarmChallengeView({Key? key}) : super(key: key);
+  AlarmChallengeView({Key? key}) : super(key: key);
+
+  final HapticFeebackService _hapticFeebackService = Get.find();
+
+  void _hapticFeedback() {
```

Comment:
Same as mentioned earlier, probably best to move this to the controller and access it via the controller in other files as well